### PR TITLE
Add jQuery.Flipster@1.1.3 w/ git auto-update

### LINKED ISSUE #12024: [Request] Add jQuery Flipster
**Library name:** jQuery Flipster
**Library description:** 
**Git repository url:** https://github.com/drien/jquery-flipster
**npm package name or url** (if there is one): https://www.npmjs.com/package/jquery.flipster
**License (List them all if it's multiple):** MIT
**Official homepage:** http://brokensquare.com/Code/jquery-flipster/demo/
**Wanna say something? Leave message here:** 

=====================

Notes from cdnjs maintainer(please remove this section after you read it):
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/50504997-request-add-jquery-flipster?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by him2him2**
close #12024
cc @drien

Pull request for issue: #12024
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/drien/jquery-flipster
 * Official website (optional, not the repository): http://brokensquare.com/Code/jquery-flipster/demo/
 * NPM package url (optional): https://www.npmjs.com/package/jquery.flipster
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 40
   - Count of stars: 510
   - Count of forks: 195
 * NPM download stats (optional):
   - Downloads in the last day: 41
   - Downloads in the last week: 145
   - Downloads in the last month: 463

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by drien** [comment]
Thanks for doing this and cc'ing me @him2him2! 👍

We can add a note about it in the flipster readme once it's merged.

**Comment by maruilian11** [comment]
@him2him2 Any ref if the npm package link is official?

**Comment by him2him2** [comment]
@maruilian11 I made the requested changes and pushed them. Please let me know if you see anything else.

Thanks for your help with this,
Ron

**Comment by him2him2** [comment]
@maruilian11
This should be sorted. Thanks again!

**Comment by maruilian11** [comment]
@him2him2 I modify your commit msg because the first line of it should be wrapped in 50 character.
And the body should be wrapped in 72. Other part is great! Thank you.

File: ajax/libs/jquery.flipster/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "+(js|css)/*.+(js|css|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.flipster/package.json]: Can you tell me what's the pattern for? Like which file in which version, as I didn't see it work in the latest version, and I assume it should at least fetched the non-minified js src there?

File: ajax/libs/jquery.flipster/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "jquery",
+    "carousel",
+    "flipster",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:drien/jquery-flipster.git"
```
Comment (sufuf3) [inline ajax/libs/jquery.flipster/package.json]: This URL can't access.
Let's revise to `git://github.com/drien/jquery-flipster.git`
Or follow the upstream git://github.com:drien/jquery-flipster.git
Thanks.

**Comment by sufuf3** [comment]
@him2him2 thanks for contribution.
Would you mind working on this and helping improve this PR?
If you have any question, please let us know.

Many thanks.

**Comment by sashberd** [comment]
@sufuf3 I updated this PR. FYR

File: ajax/libs/jquery.flipster/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/drien/jquery-flipster.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
```
Comment (extend1994) [inline ajax/libs/jquery.flipster/package.json]: For the record, line 26-29 is for v1.0.0 ~ v1.1.3 (the latest version).

File: ajax/libs/jquery.flipster/package.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      {
+        "basePath": "src",
+        "files": [
+          "+(js|css)/*.+(js|css|map)",
+          "jquery.flipster*.+(js|css|map)"
```
Comment (extend1994) [inline ajax/libs/jquery.flipster/package.json]: This is for v1.0.0 ~ v1.1.3 (the latest version).
In my view, `jquery.flipster*` is enough.

File: ajax/libs/jquery.flipster/package.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "jquery.flipster*.+(js|css|map)"
```
Comment (extend1994) [inline ajax/libs/jquery.flipster/package.json]: This is for v0.1.0/0.1.3.

File: ajax/libs/jquery.flipster/package.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "+(js|css)/*.+(js|css|map)",
```
Comment (extend1994) [inline ajax/libs/jquery.flipster/package.json]: This is for v0.2.0 ~ v0.3.5. I think `+(js|css)/jquery.flipster*` is a better option.

**Comment by sashberd** [comment]
@extend1994  The PR was updated. FYR

**Comment by sashberd** [comment]
Moved to #13692

**Pull request closed without merging**